__tests__/integration/app.test.js

```diff
@@ -0,0 +1,13 @@
+const request = require("supertest");
+const { app } = require("../../app");
+
+describe("base app behaviour", () => {
+  test("GET: 404 returns status 404 and a message when requesting an endpoint that doesn't exist", () => {
+    return request(app)
+      .get("/BadEndPoint")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+});
```

Comment:
Nice 404 test, just be aware that having jest tests in multiple files can cause issues (tests erratically failing), if you have issues down the line - consider putting all tests into one file